install_config/web_console_customization.adoc

```diff
@@ -922,3 +921,28 @@ xref:../install/index.adoc#install-planning[inventory file]:
 # See: https://docs.openshift.com/enterprise/latest/install_config/aggregate_logging.html
 #openshift_master_logging_public_url=https://kibana.example.com
 ----
+
+[[changing-the-web-console-url-port-and-certificates]]
+== Changing the Web Console URL Port and Certificates
+
+To ensure your custom certificate is served when users access the web console
+URL, add the certificate and URL to the `NamedCertificates` section of the
```

Comment:
typo `NamedCertificates` vs `namedCertificates` ?

 as per the [code](https://github.com/openshift/openshift-ansible/blob/006d82982d1dd3117bf9057ae6172a9d16380e0c/roles/openshift_control_plane/templates/master.yaml.v1.j2#L193)